src/test/java/no/sikt/nva/scrapers/DublinCoreScraperTest.java

```diff
@@ -185,10 +192,38 @@ var record = dcScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(
         assertThat(actualContributors, is(equalTo(expectedContributors)));
     }
 
+    @Test
+    void shouldExtractContributorsInRightOrderAndInjectSequnceNumber() {
+        var dc = DublinCoreFactory.createDublinCoreFromXml(new File(TEST_RESOURCE_PATH + "dc_with_contributors.xml"));
+        var record = dcScraper.validateAndParseDublinCore(dc, new BrageLocation(null), SOME_CUSTOMER);
+
+        var expectedContributors = Arrays.asList(
+            Map.entry("Jo Rune Ugulen", 1),
+            Map.entry("Tone Merete Bruvik", 2),
+            Map.entry("Rannveig Fluge", 3),
+            Map.entry("Håkon Haugland", 4),
+            Map.entry("Geir Atle Ersland", 5),
+            Map.entry("Arne Solli", 6)
```

Comment:
This is the order xml values are places in dublin core and appearance in brage gui:

```
  <dcvalue element="contributor" qualifier="none" language="eng">Ugulen, Jo Rune</dcvalue>
  <dcvalue element="contributor" qualifier="none" language="eng">Bruvik, Tone Merete</dcvalue>
  <dcvalue element="contributor" qualifier="none" language="eng">Fluge, Rannveig</dcvalue>
  <dcvalue element="contributor" qualifier="none" language="eng">Haugland, Håkon</dcvalue>
  <dcvalue element="contributor" qualifier="editor" language="eng">Ersland, Geir Atle</dcvalue>
  <dcvalue element="contributor" qualifier="editor" language="eng">Solli, Arne</dcvalue>
```

<img width="868" alt="image" src="https://github.com/BIBSYSDEV/nva-brage-migration/assets/107109909/09c70136-3212-413d-b862-b788fecc1d5b">